source/posix/file_utils.c

```diff
@@ -34,3 +37,8 @@ struct aws_string *aws_get_home_directory(struct aws_allocator *allocator) {
 
     return NULL;
 }
+
+bool aws_does_path_exist(const char *path) {
```

Comment:
we're sure we want to be using c-str instead of byte_cursor?
I can see it both ways, just want to make sure we're sure

Reply:
it would have to be null terminated, so aws_string would need to be used. At that point, I'd prefer const char * and let the user make the allocation decision.